src/Workspaces/Core/Portable/Editing/DeclarationModifiers.cs

```diff
@@ -26,7 +26,8 @@ private DeclarationModifiers(Modifiers modifiers)
             bool isWithEvents = false,
             bool isPartial = false,
             bool isAsync = false,
-            bool isWriteOnly = false)
+            bool isWriteOnly = false,
+            bool isExtern = false)
```

Comment:
❓ Is this change needed? It seems like this parameter is unused. I noticed, for example, that there is no `isRef` parameter either.